src/RolloutScorer/RolloutScorer/Commands.cs

```diff
@@ -69,7 +69,7 @@ public async Task<int> InvokeAsync(IEnumerable<string> arguments)
             _rolloutScorer.GithubConfig = StandardConfig.DefaultConfig.GithubConfig;
 
             // If they haven't told us to upload but they also haven't specified a repo & rollout start date, we need to throw
-            if (string.IsNullOrEmpty(_rolloutScorer.Repo) || (_rolloutScorer.RolloutStartDate == null))
```

Comment:
@jonfortescue Is this change ok?

Reply:
No, the second conditional us necessary for checking if a rollout start date hasn't been provided

Reply:
It is not, because RolloutStartDate is a DateTimeOffset, which cannot be null.  It was a warning from the compiler that this code was pointless, so I removed it to fix the warning.

Reply:
Is there a value that is the default for when a DateTimeOffset isn't set?

Reply:
I think the appropriate change here is to compare to DateTimeOffset.MinValue

Reply:
Would `_rolloutScorer.RolloutStartDate == default` also work?

Reply:
I'd rather not change existing behavior.  The existing behavior is as I left the code (it never could return true, so I removed the comparison).  I'd rather not mix a bunch of bug fixes into this already sadly huge change. :-(

Reply:
I hear that, but the comment suggests that the intention was to do something if the start date wasn't set. It's a bug and the code doesn't match the comment. So, IMO, you should either change it and remove the comment/change the comment so we go back and fix it, or just change it so it matches the comment and does what was intended.

Reply:
I'll delete the comment then.

Reply:
We shouldn't just fix all bugs in every PR... especially PR's that are, ostensibly, not intended to have any behavioral changes. I'm not saying there _isn't_ a bug... I'm just saying this PR isn't the appropriate place to fix said bug.  I'll open an issue to deal with it.

Reply:
Created https://github.com/dotnet/arcade/issues/9206